core/src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -199,8 +199,10 @@ private[query] class EventsByTagPublisher(
         context.system.scheduler.scheduleOnce(settings.eventualConsistencyDelay, self, Continue)
   }
 
-  def timeToLookForDelayed: Boolean =
-    strictBySeqNumber && !backtracking && lookForDelayedDeadline.isOverdue() && !isTimeBucketBeforeToday()
+  def timeToLookForDelayed: Boolean = {
+    strictBySeqNumber && toOffset.isEmpty && !backtracking &&
+      lookForDelayedDeadline.isOverdue() && !isTimeBucketBeforeToday()
+  }
```

Comment:
Hm, so if it is a `currentEventsByTag` `toOffset` will contain the end timestamp and this always return `false? Not sure how this relates to the other changes.